frontend/src/app/modules/projects/project-card/project-card.component.html

```diff
@@ -1,5 +1,25 @@
-<mat-card>
-    <mat-card-content>
+<div
+    class="project-card"
+    [ngClass]="{
+        postgreSql: project.engine === 'PostgreSql',
+        sqlServer: project.engine === 'SqlServer'
+    }">
+    <div class="side-panel"></div>
+    <div class="engine-logo-container">
+        <img [src]="engineLogoImage" alt="Engine Logo" class="engine-logo" />
+    </div>
+    <div class="project-name">
         {{ project.name }}
-    </mat-card-content>
-</mat-card>
+    </div>
+    <div class="project-tag-container">
+        <div class="project-tag">
+            {{ project.engine }}
```

Comment:
If a project has tags, they will be displayed here
![image](https://github.com/BinaryStudioAcademy/bsa-2023-squirrel/assets/139565128/e5fd80e5-ada6-4991-9407-20f44b209fbf)